src/systemctl/systemctl-logind.c

```diff
@@ -432,6 +432,12 @@ int help_boot_loader_entry(void) {
         sd_bus *bus;
         int r;
 
+        /* This is called without checking runtime scope and bus transport like we do in parse_argv().
```

Comment:
shouldn't the combination be rejected instead?

Reply:
This is just for help command, and provide some suggestions. I think it is not necessary to handle transport or scope options strictly.

Reply:
yeah but it gives the impression that it's a valid combination, which usually doesn't happen?

Reply:
The `--boot-loader-entry=` positional argument sets `arg_boot_loader_entry`, and it is only used by `halt`, `poweroff`, and `reboot` commands. They are only supported by system scope.

So, even if a user get list of entries by `systemctl --user --boot-loader-entry=help`, we will refuse the operation by `systemctl --user --boot-loader-entry=foo reboot` with sufficient error message. Note the refusal is _not_ caused by `--boot-loader-entry=foo`, but by calling `reboot` with `--user`. And, obtained boot entries themselves obtained with adjusted scope and bus transport _are_ correct.

Reply:
Upgrading the green label again. But feel free to downgrade it if you still do not like the current approach.